broker-cli/commands/wallet.js

```diff
@@ -318,20 +318,28 @@ async function release (args, opts, logger) {
     if (!ACCEPTED_ANSWERS.includes(answer.toLowerCase())) return
 
     try {
-      await client.walletService.releaseChannels({ market, force })
-    } catch (e) {
-      logger.error(`Failed to release payment channels for ${market}`.red)
+      const { errors } = await client.walletService.releaseChannels({ market, force })
+
+      if (errors) {
+        logger.info(`Errors have occurred while trying to release channels for ${market}:`.red)
+        errors.forEach(e => logger.info(e))
+        logger.info('')
+      }
 
-      if (!force) {
-        logger.info('You can force the release of funds using the `--force` option, however')
-        logger.info('this has the potential to lock your funds for an extended period of time')
-        logger.info('and cost additional fees')
+      if (errors && !force) {
+        logger.info('If the error above suggests an uncooperative closing of any channels on')
```

Comment:
[nit] consider using a multi-line string here

Reply:
@treygriffith the formatting is easier w/ individual lines so I left it

ex:
```
if (error) {
  logger.info('without weird spacing')
}
```
vs
```
if (error) {
  logger.info(`
with weird spacing
  `)
}